rllib/algorithms/ppo/ppo_catalog.py

```diff
@@ -79,7 +80,8 @@ def __init__(
             hidden_layer_dims=post_fcnet_hiddens,
             hidden_layer_activation=post_fcnet_activation,
             output_activation="linear",
-            output_dim=pi_output_dim,
+            output_dim=None,  # We don't know the output dimension yet, because it
```

Comment:
can't you just say `output_dim=self.action_dist_cls.required_model_output_shape(
                space=self.action_space, model_config=self.model_config_dict` here?

Reply:
We don't know the framework yet. So we'd have to instantiate either of the framework-specific classes here, which I guess is a no-no.

Reply:
what is framework specific? `self.action_dist_cls` is just the class.